### ISSUE TITLE
 Venator Altum adds amber when damage is blocked by armor

### ISSUE THREAD

**Issue by Why-Not-0** [keyteki/keyteki#2453]
When I seeker needle a Venator Altum and the tide is low it exalts itself

**Comment by Tutenkharnage** [comment]
FWIW, the glossary entry for "armor" states, "Armor prevents damage before it is actually dealt."

**Comment by blinkingline** [comment]
FFG definitely needs to clean up some of the language around damage, but I believe that unless a damage token would be placed on [Venator Altum](https://archonarcana.com/Venator_Altum), then it has not been dealt damage.

**Comment by UltraUltraUltra** [comment]
I agree that FFG needs to make these things clearer, but I would suggest in this situation, similar to poison, no damage has been dealt to the creature if armour prevents that damage.

It would be really helpful if FFG defined a clear distinction between damaged be assigned/attempted and damage being dealt.
